src/main/java/org/pesmypetcare/webservice/builders/DocumentPath.java

```diff
@@ -256,60 +257,18 @@ private static StringBuilder buildPathToFreqTraining(@NonNull String userId, @No
     }
 
     /**
-     * Builds the path to a medication entry.
+     * Builds the path to a pet collection entry.
      *
      * @param userId The user ID
      * @param petName The pet name
-     * @param date The date of creation
-     * @param medicationName The medication name
-     * @return The path
-     */
-    @NonNull
-    private static StringBuilder buildPathToMedication(@NonNull String userId, @NonNull String petName,
-                                                       @NonNull String date, @NonNull String medicationName) {
-        return buildPathToPet(userId, petName).append("/medications/").append(date).append("½").append(medicationName);
-    }
-
-    /**
-     * Builds the path to a frequency of wash entry.
-     *
-     * @param userId The user ID
-     * @param petName The pet name
-     * @param date The date of creation
-     * @return The path
-     */
-    @NonNull
-    private static StringBuilder buildPathToFreqWash(@NonNull String userId, @NonNull String petName,
-                                                     @NonNull String date) {
-        return buildPathToPet(userId, petName).append("/washes/").append(date);
-    }
-
-    /**
-     * Builds the path to a weight entry.
-     *
-     * @param userId The user ID
-     * @param petName The pet name
-     * @param date The date of creation
-     * @return The path
-     */
-    @NonNull
-    private static StringBuilder buildPathToWeight(@NonNull String userId, @NonNull String petName,
-                                                   @NonNull String date) {
-        return buildPathToPet(userId, petName).append("/weights/").append(date);
-    }
-
-    /**
-     * Builds the path to a meal entry.
-     *
-     * @param userId The user ID
-     * @param petName The pet name
-     * @param date The date of creation
+     * @param collectionName The pet collection name
+     * @param key Key to the collection entry
      * @return The path
      */
     @NonNull
-    private static StringBuilder buildPathToMeal(@NonNull String userId, @NonNull String petName,
-                                                 @NonNull String date) {
-        return buildPathToPet(userId, petName).append("/meals/").append(date);
+    private static StringBuilder buildPathToPetCollection(@NonNull String userId, @NonNull String petName,
+                                                          @NonNull String collectionName, @NonNull String key) {
+        return buildPathToPet(userId, petName).append("/").append(collectionName).append("/").append(key);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String "/" appears 2 times in the file.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5482446)